src/HttpClient/Guzzle6HttpClient.php

```diff
@@ -4,9 +4,9 @@
 
 use Crell\ApiProblem\ApiProblem;
 use eLife\ApiSdk\Exception\ApiException;
-use eLife\ApiSdk\Exception\ApiProblemException;
-use eLife\ApiSdk\Exception\NetworkException;
-use eLife\ApiSdk\Exception\ResponseException;
+use eLife\ApiSdk\Exception\ApiProblemResponse;
+use eLife\ApiSdk\Exception\NetworkProblem;
+use eLife\ApiSdk\Exception\BadResponse;
```

Comment:
With our `.php_cs` it seems that it doesn't:
```
$ ../proofreader-php/vendor/bin/php-cs-fixer --config-file=.php_cs fix src
Loaded config from ".php_cs"
Fixed all files in 1.256 seconds, 6.000 MB memory used
$ git diff
(empty output)
```

Maybe it's because:
`The "contrib level" fixers cannot be enabled via this option; you should instead set them manually by their name via the --fixers option.`
and we only have `-empty_return` in there?